DynamoLeagueBlazor/Server/Models/Player.cs

```diff
@@ -23,10 +23,10 @@ public Player(string name, string position, string headShotUrl)
     public ICollection<Fine> Fines { get; private set; } = new HashSet<Fine>();
 
     // TODO: State Machine these
-    public Player SetToRostered(DateTime contractedToDate, int contractValue)
+    public Player SetToRostered(int contractedToDate, int contractValue)
```

Comment:
I think renaming this is in order since it’s no longer a date. Maybe contractedToYear?